actionview/lib/action_view/template/error.rb

```diff
@@ -135,7 +136,11 @@ def corrections
           end
         end
 
-        results.to_a
+        if @error.partial
+          results.to_a.map { |res| res.gsub(/\/_/, "/") }
```

Comment:
I don't think it's likely to happen, but maybe this regex should look for just the last `/` in a string in case there was a template named ex. `foo/_bar/_baz.html.erb`?

Reply:
Updated! Thanks for catching that.